.github/workflows/gosec.yml

```diff
@@ -1,24 +0,0 @@
-name: Run Gosec
```

Comment:
Did you mean to kill this file?

Reply:
Yes, because we run gosec via golintci now. This avoids duplication *and* it allows us to add skiprules in a unified way (e.g. via //nolint: comments)